docs/source_values.rst

```diff
@@ -178,12 +181,106 @@ In this example, the LED is lit only when the button is not pressed:
 
 Some tools combine the values of multiple sources:
 
-.. image:: images/combining_sources.*
+.. image:: images/source_values/combining_sources.*
     :align: center
 
 In this example, the LED is lit only if both buttons are pressed (like an
 `AND`_ gate):
 
+.. image:: images/source_values/combining_sources_led_buttons.*
+    :align: center
+
 .. literalinclude:: examples/combining_sources.py
 
 .. _AND: https://en.wikipedia.org/wiki/AND_gate
+
+While most devices have a :attr:`~Device.value` range between 0 and 1, some have
+a range between -1 and 1 (e.g. :class:`Motor`, :class:`Servo` and
+:class:`TonalBuzzer`). Some source tools output values between -1 and 1, which
+are ideal for these devices, for example passing :func:`~tools.sin_values` in:
+
+.. image:: images/source_values/sin_values.*
+    :align: center
+
+.. literalinclude:: examples/sin_values.py
+
+In this example, all three devices are following the `sine wave`_. The motor
+ramps up from 0 (stopped) to 1 (full speed forwards), then back down to 0 and on
+to -1 (full speed backwards) in a cycle. Similarly, the servo moves from its mid
+point to the right, then towards the left; and the buzzer starts with its mid
+tone, gradually raises its frequency, to its highest tone, then down towards its
+lowest tone. Note that setting :attr:`~SourceMixin.source_delay` will alter
+the speed at which the device iterates through the values.
+
+.. _sine wave: https://en.wikipedia.org/wiki/Sine_wave
+
+Internal devices
+----------------
+
+GPIO Zero also provides several :doc:`internal devices <api_internal>` which
+represent facilities provided by the operating system itself. These can be used
+to react to things like the time of day, or whether a server is available on the
+network. These classes include a :attr:`~ValuesMixin.values` property which can
+be used to feed values into a device's :attr:`~SourceMixin.source`. For example,
+a lamp connected to an :class:`Energenie` socket can be controlled by a
+:class:`TimeOfDay` object so that it is on during the hours of 8am and 8pm:
+
+.. image:: images/source_values/timed_heat_lamp.*
+    :align: center
+
+.. literalinclude:: examples/timed_heat_lamp.py
+
+Using the :class:`DiskUsage` class with :class:`LEDBarGraph` can show your Pi's
+disk usage percentage on a bar graph:
+
+.. image:: images/source_values/disk_usage_bar_graph.*
+    :align: center
+
+.. literalinclude:: examples/disk_usage_bar_graph.py
+
+Demonstrating a garden light system whereby the light comes on if it's dark and
+there's motion is simple enough, but it requires using the
+:func:`~tools.booleanized` source tool to convert the light sensor from a float
+value into a boolean:
```

Comment:
Ahhh, this explains my earlier comment :wink: I'd forgotten that the light sensor returns a continuous 0->1 value and not a boolean 0 or 1 value.

Although it raises an interesting point for future discussion - for those devices like LightSensor that have an `is_active` property, I wonder if there'd be any easy way to use `is_active` (boolean) with source-tools instead of `value` (float) ?

Reply:
I suggested that in #589 and you disagreed! :P

Reply:
LOL, I'd completely forgotten about that! :joy: 
Looks like I _didn't_ disagree though...
> But yeah, for those devices which have an 'implicit' boolean state like `light_detected` it _might_ be nice to offer a simpler way to use that with source-values.
Instead of naming it `bool_values` what about `active_values`? So that `values` returns an infinite generator of `value`, and `active_values` returns an infinite generator of `is_active`?

Why was that issue closed?

Reply:
I think I agreed with you that using `booleanized` was easy enough and avoiding API bloat. Also trying to keep the number of unresolved issues down, closing any I think are dead suggestions.

Reply:
Hmmm, looking at https://gpiozero.readthedocs.io/en/master/api_input.html it seems both MotionSensor and LightSensor inherit from SmoothedInputDevice - does that mean that they both have a float-value, and so **both** need to be booleanized?

Reply:
@lurch I've figured out why they're different. `MotionSensor` has a default `queue_len` of 1 and `LightSensor` has a default `queue_len` of 5, therefore LS's value is averaged and MS's effectively isn't.

However, setting MS's `queue_len` to >1 gives it a boolean value:

```python
>>> pir = MotionSensor(14)
>>> pir.value
0
>>> pir = MotionSensor(15, queue_len=2)
>>> pir.value
0.0
```

Reply:
Yup, see my comment here https://github.com/RPi-Distro/python-gpiozero/pull/744#discussion_r256206318